### ISSUE TITLE
Direct npx usage support

### ISSUE THREAD

**Issue by privatenumber** [skypackjs/package-check#8]
I noticed both these package names are available:
- https://www.npmjs.com/package/package-check
- https://www.npmjs.com/package/pkg-check

Can it be published without the npm org so it can be directly invoked without installation? eg.

```sh
npx pkg-check
```

**Comment by gr2m** [comment]
`npx @skypack/package-check` should work just fine

**Comment by gr2m** [comment]
Maybe we should mention it in the README @FredKSchott?

**Comment by privatenumber** [comment]
Yeah it does. The brevity of a command without the org is nice though.
